lib/processors/manifestCreator.js

```diff
@@ -429,11 +385,6 @@ async function createManifest(libraryResource, libBundle, descriptorVersion, _in
 
 	function createSapUI5() {
 		function getUI5Version() {
-			let ui5Version;
-			if ( ui5Version != null ) {
-				return ui5Version;
-			}
```

Comment:
I wonder where this code came from. It doesn't make sense, but maybe it needs to be fixed rather than being removed.

Reply:
The code taken from the Java implementation had a member variable ui5version which could be set from the oustide via parameter. This is not the case here anymore so it can be removed.